src/client/NativeClient.js

```diff
@@ -56,17 +56,17 @@ export default class NativeClient extends Client {
                 switch (props.network) {
                     case "mainnet":
                         this.setNetwork(Network.MAINNET);
-                        this.setNetworkName("mainnet");
+                        this.setNetworkName(NetworkName.NETNAMES[0]);
```

Comment:
I think if you use `NetworkName.MAINNET` it might be better since `NETNAME[0]` isn't very descriptive.